src/levanter/models/flash_attention.py

```diff
@@ -68,22 +68,50 @@ def flash_attention(
     KPos = k.resolve_axis(KPos)
 
     if QPos.size < block_size or KPos.size < block_size:
-        from levanter.layers.attention import simple_attention_with_dropout
-
-        return simple_attention_with_dropout(
-            QPos,
-            KPos,
-            Key,
-            q,
-            k,
-            v,
-            mask=mask,
-            bias=bias,
-            dropout=dropout,
-            inference=inference,
-            prng=key,
-            logits_soft_cap=logits_soft_cap,
-        )
+        if attn_sink is not None:
+            from levanter.layers.attention import dot_product_attention_with_sink
+
+            return dot_product_attention_with_sink(
+                QPos,
+                KPos,
+                Key,
+                q,
+                k,
+                v,
+                attn_sink,
+                mask=mask,
+                bias=bias,
+                attention_dtype=dtype,
+                precision=precision,
+                use_flash=False,  # force VANILLA
```

Comment:
can we not call dot_product_attention_with_sink in the attn_sink is None route too?